sp800-63a/sec6_dc.md

```diff
@@ -2,64 +2,46 @@
 
 <div class="breaker"></div>
 
-# 6. Leveraging Antecedent Proofing Events
+# <a name="derived-authN"></a> 6. Derived Authenticators
 
 _This section is normative._
 
-Possession of an authenticator, bound to an identity proofed at a given IAL, should allow a claimant to obtain additional authenticators, per CSP policy.  This section details requirements for two specific use cases:
+The expression of IAL and AAL as separate ordinals allows organizations to issue any type or number of authenticators they want regardless of how or if a subject is identity proofed.  This is one of the benefits of the separation - an organization can issue authenticators suitable for use by subscribers regardless of the required IAL to access the service.  However, there are instances when a CSP will want to bind a new authenticator to an existing, identity proofed record, or credential. Since it is beneficial to the subject and CSP to minimize the number of times the identity proofing process needs to be repeated, this can be accomplished by proving possession and successful authentication of an existing authenticator that is already bound to a proofed digital identity. Authenticators issued and bound to existing identity proofed records are considered to be *derived authenticators*.
 
-1. A _claimant_ obtains a secondary, or derived authenticator, for use only within the limits and authorizations of the primary authenticator.  For example, this could be a derived PIV credential in federal use cases, or a temporary authenticator. See [Section 6.1](#dc) for more details.
-2. An _applicant_ seeks to obtain another primary authenticator from a CSP that did not issue the original authenticator. For example, if an applicant wants to switch CSPs and/or have another authenticator at a new CSP for other uses (e.g. basic browsing vs. financial). See [Section 6.2](#prior) for more details.
+The definition of derived in this section does *not* infer that an authenticator is cryptographically tied to a primary authenticator, for example deriving a key from another key.  Rather, the derived authenticator is issued and bound to an identity that has already been identity proofed.  To prove that identity proofing has occurred, the individual demonstrates possession and authentication of an authenticator bound to the original proofed identity record (i.e the credential). 
```

Comment:
"has already been identity proofed" -> "has already been registered by the CSP" (again emphasizing that we're binding to an existing record, not just being efficient)

"authentication of an authenticator" -> "authentication using authenticator(s)"